Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
Stranger and stranger - my "Monokai" theme doesn't look anything like that!

(on OS X)
<img width="399" alt="screen shot 2016-04-18 at 9 42 29 pm" src="https://cloud.githubusercontent.com/assets/27570/14627109/7c0dd6cc-05ae-11e6-9b6a-ec6bc686f9e0.png">

If you put your cursor before the pink colored whitespace, what scope names do you see? (control+shift+P, or Tools > Developer > Show Scope Name)